proof/invariant-abstract/DetSchedSchedule_AI.thy

```diff
@@ -7581,17 +7603,18 @@ lemma sc_valid_refills_times_bounded:
   \<Longrightarrow> unat (r_time (refill_hd sc)) + unat (sc_period sc) \<le> unat max_time
   \<Longrightarrow> x \<in> set (sc_refills sc)
   \<Longrightarrow> r_time x \<le> r_time (refill_hd sc) + sc_period sc"
-  apply (frule (1) sc_valid_refills_times_bounded_helper)
+  apply (frule (2) sc_valid_refills_times_bounded_helper)
   apply (clarsimp simp: sc_valid_refills_def window_def word_le_nat_alt split: if_splits)
   apply (subst unat_add_lem', overflow_hammer)
   apply (erule (1) order_trans[rotated])
   done
 
 lemma sc_valid_refills_no_overflow:
   "sc_valid_refills sc
```

Comment:
I've seen Gerwin say elsewhere that we prefer the [| _ ; _; ... |] ==> _ format, rather than the string of ==>s (though I'm not certain of his reasons)

Reply:
I changed a few and then realised how many might need changing so I left it at one or two fix-ups.